_data/companies.yml

```
@@ -2293,3 +2293,10 @@
   visitors: Discouraged
   events: Discouraged 
   last_update: 2020-03-12
+
+- name: UWorld LLC
+  wfh: required
```

```suggestion
  wfh: Required
```

Fixing case for consistency